### PR TITLE
Function cholesky inverse

### PR DESCRIPTION
**Pull request by Rounak183**
Function and testing code for cholesky_inverse functionality for the PyTorch frontend API.

### CODE REVIEW

**Comment by Rounak183** [comment]
@paulaehab Please review my code and let me know the changes!

**Comment by paulaehab** [comment]
Close #15683

**Comment by paulaehab** [comment]
Hey @Rounak183 🙂, I will link this to the corresponding issue, but in future you should do this yourself by adding a comment ‘Close #Issue_number’, as explained here: (https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues)

File: ivy/functional/frontends/torch/linalg.py [hunk 1]
```diff
@@ -132,6 +132,17 @@ def cholesky(input, *, upper=False, out=None):
     return ivy.cholesky(input, upper=upper, out=out)
 
 
+@to_ivy_arrays_and_back
```
Comment (paulaehab) [inline ivy/functional/frontends/torch/linalg.py:135]: Hi @Rounak183 this implementation must be in blas_and_lapack_ops.py file not in linalg.py that is why all tests not running so please code it in it's file and then try to make all tests pass

**Review by paulaehab — COMMENTED** [review]
Please implement the function in blas_and_lapack_ops.py not in linalg.py and be sure all tests are passed

**Review by paulaehab — APPROVED** [review]
Good work, now you have to be sure that the function pass all tests

**Comment by Rounak183** [comment]
> Good work, now you have to be sure that the function pass all tests

It's failing 3 tests. Would require some help from you. @paulaehab 
The 16th one has this error -  -0.10649996250867844!=-0.10649752616882324. Difference in result from tensorflow framework and ground truth framework. How do I resolve such minute results?

**Comment by Rounak183** [comment]
Hey @paulaehab. Need to just fix 1 test case. Can't seem to figure out where I'm going wrong. The failing test says exceeding maximum recursion limit.

File: ivy/functional/ivy/linear_algebra.py [hunk 2]
```diff
@@ -160,6 +160,22 @@ def cholesky(
     return current_backend(x).cholesky(x, upper=upper, out=out)
```
Comment (paulaehab) [inline ivy/functional/ivy/linear_algebra.py:162]: Hello, you are adding frontend function you just can't edit or in ivy/functional you must add the function in frontend only as the cholesky_inverse is not common function in all frameworks you have to implement it's functionality in frontend

**Review by paulaehab — COMMENTED** [review]
@Rounak183 sorry for being too late the PR somehow slipped under the radar.

File: ivy/functional/ivy/linear_algebra.py (hunk 2 above)
Comment (paulaehab) [reply ivy/functional/ivy/linear_algebra.py:162]: Also you can check in the documentation the concept of adding frontend function (https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html#ivy-frontends)

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
